package.json

```diff
@@ -52,21 +52,33 @@
     "prop-types": "^15.7.2",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
-    "webpack": "^4.35.2",
+    "webpack": "^4.35.3",
```

Comment:
@jennizalwango I didn't upgrade this intentionally but it might be updated by other packages. not really sure why it changed. I guess it ok we leave it to the updated version.

Reply:
okay